README.rst

```diff
@@ -26,6 +26,31 @@ Please ensure all dependencies are available on the CircuitPython filesystem.
 This is easily achieved by downloading
 `the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
 
+Installing from PyPI
+--------------------
+
+On supported GNU/Linux systems like the Raspberry Pi, you can install the driver locally `from
+PyPI <https://pypi.org/project/adafruit-circuitpython-bme280/>`_. To install for current user:
```

Comment:
Please replace `bme280` with `tinylora`.

Reply:
Replaced, was focusing on making sure the license info was correct and missed this (added after)